packages/govern-core/contracts/Govern.sol

```diff
@@ -21,7 +21,7 @@ contract Govern is AdaptativeERC165, IERC3000Executor, ACL {
 
     event ETHDeposited(address indexed sender, uint256 value);
 
-    constructor(address _initialExecutor) ACL(address(this)) public {
+    constructor(address _initialExecutor) ACL(address(_initialExecutor)) public {
```

Comment:
This was actually intentional. The root of Govern should be Govern itself.

This means that Govern itself (via action execution) can change the ACL in order to add other executors. The initial executor should have execution granted but it shouldn't be root IMO

Reply:
See https://github.com/aragon/govern/pull/257